pkg/testframework/wait.go

```diff
@@ -31,7 +31,7 @@ func WaitHTTP(rt http.RoundTripper, url string) error {
 	httpClient := &http.Client{
 		Transport: rt,
 	}
-	err := wait.Poll(500*time.Millisecond, wait.ForeverTestTimeout, func() (done bool, err error) {
+	err := wait.Poll(500*time.Millisecond, time.Second*120, func() (done bool, err error) {
```

Comment:
mildly interesting.  "ForeverTestTimeout" is 30s.  I found it wasn't high enough when running locally, at least...possibly due to image pulls.

Reply:
https://en.wikipedia.org/wiki/Forever_May_Not_Be_Long_Enough